source/brailleTables.py

```diff
@@ -77,6 +111,8 @@ def addTable(
 	output: bool = True,
 	input: bool = True,
 	source: str = TableSource.BUILTIN,
+	inputForLangs: set[str] | None = None,
+	translationForLangs: set[str] | None = None,
```

Comment:
As above, I'd use the word output here as well to keep consistency.

```suggestion
	outputForLangs: set[str] | None = None,
```